spec/unit/generator/embed_frameworks_script_spec.rb

```diff
@@ -4,7 +4,7 @@ module Pod
   describe Generator::EmbedFrameworksScript do
     it 'returns the embed frameworks script' do
       frameworks = {
-        'Debug' => [Target::FrameworkPaths.new('Pods/Loopback.framework', 'Pods/Loopback.framework.dSYM'),
+        'Debug' => [Target::FrameworkPaths.new('Pods/Loopback.framework', 'Pods/Loopback.framework.dSYM', ['7724D6B4-C7DD-31F0-80C6-EE818ED30B07.bcsymbolmap']),
```

Comment:
Could you have this include more than one symbol map to test that it creates a call to `install_bcsymbolmap` for each one?

Reply:
do you mean include a real `.bcsymbolmap` in an integration spec?

Reply:
Yes, later

Reply:
> Could you have this include more than one symbol map to test that it creates a call to `install_bcsymbolmap` for each one?

@dnkoutso Add symbol map Has been added